t/FHEM/10_BindingsIo/00_load.t

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env perl
+use strict;
+use warnings;
+use File::Basename;
+
+use Test2::V0;
+use Test2::Tools::Compare qw{ is };
+
+my $module = basename (dirname(__FILE__));
+
+plan(1);
+is(CommandReload(undef,$module), U(), "$module loaded");
+
+exit(0);  # necessary
+
+1;
+
```

Comment:
Wann und wo werden diese Tests ausgeführt?

Reply:
Der Workflow führt diese mit identischer Bedingung wir das Update der controls Datei durch.
Die Ergebnisse sind dann in den Actions zu sehen:

https://github.com/fhempy/fhempy/actions/runs/3338660936

Reply:
Nein, der commit geht in das zuvor ausgecheckte repository (Feature Branch)

In meinem Fork sieht das so aus:
https://github.com/sidey79/fhempy/actions/runs/3363280989/jobs/5576210726

Der commit mit der aktualisierten controls soll allerdings absichtlich in den Featurebranche gehen, damit dieser mittels Update Befehl verwendet werden kann. Das ist nicht möglich, wenn die controls Datei (wie vor dem PR) ausschließlich im master aktualisiert wird.

Was ich jetzt allerdings nicht verifiziert habe ist, was passiert, wenn im master ohne Feature-Branch gepusht wird.

Reply:
Die Nutzung eines Feature-Branch durch die User macht nur teilweise Sinn, da im Moment kein pypi Paket für Feature-Branches erstellt wird. Generell ist es aber schon vernünftig die controls im Branch zu aktualisieren und dann nach master zu mergen.

Reply:
Ich wollte die Anpassungen halt in echt verwenden und nahm an, Du möchtest dass vielleicht auch :)